docs/serving/configuration/config-defaults.md

```diff
@@ -30,319 +32,497 @@ data:
-By default, it is not set by Knative.
+The `revision-memory-request` value determines the memory allocation assigned
+to revisions by default. If this value is omitted, the system default is used. This key is not enabled by default for Knative.
 {% endraw %}
 
-**Key**: `revision-memory-request`
-
-**Default**: `"100M"` (100 megabytes of memory)
+* **Global key:** `revision-memory-request`
+* **Per-revision annotation key:** `serving.knative.dev/revision-memory-request`
+* **Possible values:** integer
+* **Default:** `"100M"` (100 megabytes of memory)
 
 **Example:**
-```yaml
-apiVersion:  v1
-kind:  ConfigMap
-metadata:
-  name:  config-defaults
-  namespace:  knative-serving
-data:
-  revision-memory-request: "100M" # 100 megabytes of memory
-```
+
+=== "Global (ConfigMap)"
+    ```yaml
+    apiVersion:  v1
+    kind:  ConfigMap
+    metadata:
+      name:  config-defaults
+      namespace:  knative-serving
+    data:
+      revision-memory-request: "100M"
+    ```
+
+=== "Per Revision"
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: helloworld-go
+      namespace: default
+    spec:
+      template:
+        metadata:
+          annotations:
+            serving.knative.dev/revision-memory-request: "400m"
+        spec:
+          containers:
+            - image: gcr.io/knative-samples/helloworld-go
+    ```
 
 ### Revision Ephemeral Storage Request
 {% raw %}
-revision-ephemeral-storage-request contains the ephemeral storage
-allocation to assign to revisions by default.  If omitted, no value is
-specified and the system default is used.
+The `revision-ephemeral-storage-request` value determines the ephemeral storage
+allocation assigned to revisions by default. If this value is omitted, the system default is used. This key is not enabled by default for Knative.
 {% endraw %}
 
-**Key**: `revision-ephemeral-storage-request`
-
-**Default**: `"500M"` (500 megabytes of storage)
+* **Global key:** `revision-ephemeral-storage-request`
+* **Per-revision annotation key:** `serving.knative.dev/revision-ephemeral-storage-request`
+* **Possible values:** integer
+* **Default:** `"500M"` (500 megabytes of storage)
 
 **Example:**
-```yaml
-apiVersion:  v1
-kind:  ConfigMap
-metadata:
-  name:  config-defaults
-  namespace:  knative-serving
-data:
-  revision-ephemeral-storage-request: "500M" # 500 megabytes of storage
-```
 
-### Revision Cpu Limit
+=== "Global (ConfigMap)"
+    ```yaml
+    apiVersion:  v1
+    kind:  ConfigMap
+    metadata:
+      name:  config-defaults
+      namespace:  knative-serving
+    data:
+      revision-ephemeral-storage-request: "500M"
+    ```
+
+=== "Per Revision"
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: helloworld-go
+      namespace: default
+    spec:
+      template:
+        metadata:
+          annotations:
+            serving.knative.dev/revision-ephemeral-storage-request: "500M"
```

Comment:
Likewise this annotation doesn't exist - the config map property configures the  default value for `Spec.Containers[*].Requests.EphemeralStorage` of a `Revision`

Reply:
similar to above but "ephemeral-storage" instead of "cpu" 

ie.  https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#setting-requests-and-limits-for-local-ephemeral-storage